ush/config_defaults.yaml

```diff
@@ -3038,22 +3047,29 @@ cpl_aqm_parm:
   AQM_RC_PRODUCT_FN: "aqm.prod.nc"
   AQM_RC_PRODUCT_FREQUENCY: "hourly"
 
-  DCOMINchem_lbcs_dfv: ""
+  FIXaqmchem_lbcs: ""
```

Comment:
You've already defined `FIXaqmchem_lbcs` above.

Reply:
@JianpingHuang-NOAA, this parameter is still duplicated. you define this below (Ln 3056). please remove this line.

Reply:
deleted it now.